### DISCUSSION TITLE
QStudio - Free SQL Client?

### DISCUSSION THREAD

**Discussion by ryanhamilton** [databendlabs/databend#16459] [Q&A]
Hi,
I'm the author of qStudio https://github.com/timeseries/qstudio  a free SQL tool specialized for data analysis, that allows easy charting, excel export and pivots. I'm looking for a Databend expert to work together with to get qStudio working well with Databend.

Ideally we will produce a guide like these but for Databend:
https://www.timestored.com/qstudio/database

I really need someones assistance to:
1. Try connecting using qStudio 
2. Try queries with most data types.
3. Report any bugs or improvements.

I will then fix all the issues and bugs. The reason we can't do the testing ourself is that we cannot be experts and know every data type etc. that exists for every database. Hence why we need help.

Let me know if interested.

**Comment by hantmac** [comment]
@ryanhamilton  `qstudio` looks like a very good SQL tool. I would like to make a pr to make Databend work in qstudio. Would you like to give me some develop docs?

**Comment by hantmac** [comment]
I see the `qstudio` is developed in java, databend provides [databend-jdbc](https://github.com/datafuselabs/databend-jdbc), maybe it can make some help.

**Comment by ryanhamilton** [comment]
Hi @hantmac  I will add support next week and send you details of a version to test if you could, that would be great.
